packages/app/src/app/pages/Sandbox/Editor/Header/PrivacyTooltip/elements.ts

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+import css from '@styled-system/css';
+
+export const Container = styled.div(
+  css({
+    '.tippy-content': {
```

Comment:
Override tippy design inside this popup. Would be nice when we can move this to common